package.json

```diff
@@ -67,6 +67,7 @@
     "npm-run-all": "^4.1.5",
     "polymer-webpack-loader": "^2.0.3",
     "prettier": "^2.2.1",
+    "prettier-plugin-organize-imports": "^2.3.4",
```

Comment:
Wasn't there an ESLint rule to do this that Vaadin used https://github.com/vaadin/web-components/pull/2915

Why wouldn't we stay as close as possible to that?

Reply:
I'll switch to `eslint-plugin-simple-import-sort`.